javascript/builtins/Intl.json

```diff
@@ -1272,7 +1272,7 @@
                 "version_added": "63"
               },
               "edge": {
-                "version_added": true
+                "version_added": "18"
```

Comment:
Confirmed by `Intl.PluralRules` being present in Edge 18 but missing in Edge 17.